Essentials/src/main/resources/messages.properties

```
@@ -232,9 +232,23 @@ destinationNotSet=Destination not set\!
 disabled=disabled
 disabledToSpawnMob=\u00a74Spawning this mob was disabled in the config file.
 disableUnlimited=\u00a76Disabled unlimited placing of\u00a7c {0} \u00a76for\u00a7c {1}\u00a76.
+discordCommandAccountArgumentUser=The discord account to check the minecraft of
```

```suggestion
discordCommandAccountArgumentUser=The Discord account to look up
```
Command description already specifies that it looks up MC usernames, and couldn't think of a less clunky way to word it here